Sources/Amplitude/AMPStorage.h

```diff
@@ -0,0 +1,36 @@
+//
+//  Amplitude.m
+//  Copyright (c) 2013 Amplitude Inc. (https://amplitude.com/)
```

Comment:
Is this year should be up to date?

Reply:
Copyrights do not have to be updated 
https://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated